src/JustSaying/Fluent/SqsReadConfigurationBuilder.cs

```diff
@@ -20,12 +21,53 @@ public sealed class SqsReadConfigurationBuilder : SqsConfigurationBuilder<SqsRea
 
         private string SubscriptionGroupName { get; set; }
 
+        private Action<HandlerMiddlewareBuilder> MiddlewareConfiguration { get; set; }
+
+        /// <summary>
+        /// Configures this read configuration to use a custom subscription group.
+        /// By default, each queue has its own subscription group.
+        /// </summary>
+        /// <param name="subscriptionGroupName">The name of the subscription group that this
+        /// configuration should be part of</param>
+        /// <returns>The current <see cref="SqsReadConfigurationBuilder"/>.</returns>
         public SqsReadConfigurationBuilder WithSubscriptionGroup(string subscriptionGroupName)
         {
             SubscriptionGroupName = subscriptionGroupName;
             return this;
         }
 
+        /// <summary>
+        /// Configures the middleware pipeline for this subscription.
+        /// Any middleware configured here will be wrapped around a handler and metrics middleware.
+        /// </summary>
+        /// <param name="middlewareConfiguration"></param>
+        /// <example>
+        /// A sample configuration:
+        /// <code>
+        /// WithMiddlewareConfiguration(pipe =>
+        /// {
+        ///     pipe.Use&lt;SomeCustomMiddleware&gt;();
+        ///     pipe.Use&lt;SomeOtherCustomMiddleware&gt;();
+        /// });
+        /// </code>
+        /// would yield this order of execution:
+        /// Before_SomeCustomMiddleware
```

Comment:
Could format this as an HTML `<ul>`?

Reply:
We could do - this is the appearance in rider(half with, half without):

![image](https://user-images.githubusercontent.com/130231/99518181-25b76880-2988-11eb-9b4d-bb64185d24d7.png)

I checked the formatting in VS and Rider, and had to change a few names to make it format correcly, but maybe a ul is a better way to go.

Reply:
Have moved this to ul for consistent formatting across ide's